app/models/region.rb

```diff
@@ -2,4 +2,9 @@ class Region < ApplicationRecord
   has_many :subprojects
 
   validates :name, presence: true, uniqueness: true
+  validates :latitude, :longitude,
+            numericality: {
+              greater_than_or_equal_to: -9999.999999,
+              less_than_or_equal_to: 9999.999999
```

Comment:
To emulate precision 10, scale 6 in the schema: 10 sigfigs, 6 decimal places at most.

Also do we need a test for this behaviour, and if so should it be in the controller tests or model tests.

Reply:
Nvm the limits on latitude and longitude are only -90 to 90 and -180 to 180 lol